cmd/api/src/api/v2/search.go

```
@@ -213,30 +217,48 @@ func resolveCollected(node *graph.Node) bool {
+				envProperties.Type = schemaEnvironment.SchemaExtensionDisplayName
+				envProperties.KindDisplayName = &schemaEnvironment.EnvironmentKindName
+				envProperties.KindId = &schemaEnvironment.EnvironmentKindId
+			}
+		}
 	}
-	if node.Kinds.ContainsOneOf(ad.Domain) {
-		return "active-directory"
+	return envProperties
+}
+
+func resolveExtensionID(node *graph.Node, kindToSchemaEnvironment model.EnvironmentKindsToEnvironment) *int32 {
```

golangci-lint (2.12.2)

[major] func resolveExtensionID is unused

(unused)